lib/input/bloblang.go

```diff
@@ -123,10 +146,40 @@ func newBloblang(conf BloblangConfig) (*Bloblang, error) {
 		exec:        exec,
 		remaining:   remaining,
 		timer:       timer,
+		schedule:    schedule,
+		location:    location,
 		firstIsFree: true,
```

Comment:
When set `true` this field ensures that a message is generated immediately regardless of the timer, but in the case where we have a cron schedule I wouldn't expect this so we should probably set it to `false` when `schedule` is non-nil.